lib/recurly/client.php

```diff
@@ -48,7 +48,7 @@ class Recurly_Client
   const PATH_ADJUSTMENTS = '/adjustments';
   const PATH_BILLING_INFO = '/billing_info';
   const PATH_COUPON = '/coupon';
-  const PATH_COUPON_REDEMPTION = '/redemption';
+  const PATH_COUPON_REDEMPTION = '/redemptions';
```

Comment:
So looking back at the sample code that's on the docs I linked to it doesn't quite demonstrate the issue. If you try to fetch the redemption this way: 
```
$account_code = '8dbe376603a8360c85da2b494d09b5';
$redemption = Recurly_CouponRedemption::get($account_code);
```
You'll see the problem. There's no specs for the redemptions making this easy to miss. I'll try to add a test case real quick.